src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.java

```diff
@@ -33,22 +31,19 @@ private ExtremismProfile decisionProcess(final String nomsId, final boolean prev
                 .provisionalCategorisation(RiskProfile.DEFAULT_CAT);
 
         pathFinder.ifPresent(pf -> {
-            final var banding = StringUtils.upperCase(pf.getPathFinderBanding());
-            log.debug("extremism: {} in pathfinder on {}, increased Risk of Extremism", nomsId, banding);
+            final var banding = pf.getPathFinderBanding();
+            log.info("extremism: {} in pathfinder on {}, increased Risk of Extremism", nomsId, banding);
             extremism.increasedRiskOfExtremism(true);
-            if (banding.contains("BAND 1") || banding.contains("BAND 2")) {
+            if (banding == 1 || banding == 2) {
                 extremism.notifyRegionalCTLead(true);
-                log.debug("extremism: {} Increased Risk of Extremism", nomsId);
-
                 if (previousOffences) {
-                    log.debug("extremism: {} has previous offences", nomsId);
+                    log.info("extremism: {} has previous offences", nomsId);
                     extremism.provisionalCategorisation("B");
                 } else {
                     extremism.provisionalCategorisation("C");
                 }
             } else {
```

Comment:
minor but this `} else {` can be switched over to an `else  if` therefore avoiding the nested `if` block

Reply:
👍